src/main/world/3d-sprite.shader

```diff
@@ -0,0 +1,38 @@
+shader_type spatial;
+
+render_mode unshaded;
+
+//uniform texture baseAlbedo: hint_albedo;
+uniform sampler2D baseAlbedo: hint_albedo;
+uniform float width: hint_range(0.0, 16.0);
+uniform vec2 texture_offset;
+uniform vec4 black : hint_color;
+
+varying vec2 vertex_position;
```

Comment:
Comment on fields

Reply:
Added field-level comments. Renamed some fields so that they didn't need comments.